sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -19,8 +19,6 @@
 from sklearn.pipeline import make_pipeline
 from sklearn.utils._testing import assert_array_almost_equal
 from sklearn.utils._testing import assert_array_equal
-from sklearn.utils._testing import assert_raises
-from sklearn.utils._testing import assert_raises_regex
```

Comment:
The same in this file, could you replace the `assert_raise_message` by `pytest.raises`.